chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java

```diff
@@ -99,8 +180,17 @@ public void writeModelsToDisk(String dir, String modelName){
         System.out.println("Done training, models are in " + dir+File.separator+modelName+".lc (.lex)");
     }
     public static void main(String[] args) {
-        ChunkerTrain trainer = new ChunkerTrain();
-        trainer.trainModels();
-        trainer.writeModelsToDisk();
+        if(args.length!=4&&args.length!=5){
+            System.out.println("Usage1: ...ChunkerTrain traindata modeldir modelname round");
+            System.out.println("Usage2: ...ChunkerTrain traindata modeldir modelname max_round dev_ratio");
```

Comment:
After restarting semaphore for more than 5 times, it passed finally. Can you accept this pull request?